invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/components/uploads.js

```diff
@@ -255,16 +255,10 @@ export const RDMEmptyResults = (props) => {
     <Segment placeholder textAlign="center">
       <Header icon>
         <Icon name="search" />
-        {i18next.t("No results found!")}
+        {i18next.t("No uploads found!")}
```

Comment:
I see you updated the phrases but the translations files were not refreshed. Check RDM docs for more info or ping me

Reply:
Following the ["Translation"](https://inveniordm.docs.cern.ch/develop/topics/i18n/) docs I run the `npm run extract_messages`, let me know if I should do something else